server/insert_data.go

```diff
@@ -0,0 +1,52 @@
+package server
+
+import (
+	"fmt"
+	_ "github.com/lib/pq"
+	"github.com/vkhichar/asset-management/config"
+	"github.com/vkhichar/asset-management/repository"
+	"database/sql"
+	"github.com/jmoiron/sqlx"
+)
+
+
+
+func Insert_data(){
+	err := config.Init()
+	if err != nil {
+		fmt.Printf("main: error while initialising config: %s", err.Error())
+		return
+	}
+	repository.InitDB()
+
+	var d *sqlx.DB
+
+	d = repository.GetDB()
+
+	tx:=d.MustBegin()
+	tx.MustExec("Insert into users values ($1,$2,$3,$4)","Jan Doe","jandoe@gmail.com","12345",true)
```

Comment:
Specify column names in the query. this script will fail if we add more columns in the table